### PR TITLE
filtering on internal objects

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/common/validator/AbstractPaginationValidator.java [hunk 1]
```diff
@@ -79,40 +84,58 @@ public void validateRestListRequest(RestListRequest listRequest, Class<?> type)
...
     }
-    
+
     public boolean isValidField(String fieldName, Class<?> type) {
-        List<String> fields = new ArrayList<>();
+        Map<String, Field> fields = new HashMap<>();
         fields = getAllFields(fields, type);
-        return fields.contains(fieldName);
+        final Map<String, Field> fields2 = fields;
+        if (fieldName.contains(".")) {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/common/validator/AbstractPaginationValidator.java]: Add some test coverage for this.

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**